frontend/src/components/helpers/locationMarker.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
-const LocationMarker = (props) => (
-    <svg preserveAspectRatio="none" viewBox="0 0 24 24" width="32" height="32">
+const LocationMarker = ({className}) => (
```

Comment:
In previous you had spacing between the the brackets and the variables. Like I said just be consistent. Personally I prefer the spacing but it's up to your group to decide. It's trivial I'd just go with one and not change it.